### ISSUE TITLE
faster way to favorite

### ISSUE THREAD

**Issue by ghosttie** [cheeaun/phanpy#124]
Although I appreciate the cleanness of removing as much UI as possible, it's annoying to have to click twice to favorite a toot

**Comment by pipes5210** [comment]
+1,the lack of the like and boost buttons is really the biggest thing stopping me from using phanpy

**Comment by jfrantzius** [comment]
I really really like Phanpy, it's got a great UI + UX!

One thing that I do very often is to like / favorite posts in my timeline, because I want to let the authors know that their posts are appreciated. It's really a little sad that this requires extra clicks / taps 🥺

In the end, this means that the more people use Phanpy, the less authors will receive likes for their posts, as this is immediately possible at least in every Mastodon client I know, and the more clicks / taps it requires, the less it will happen ...

**Comment by marians** [comment]
I have to agree. Especially with the importance of boosts in the fediverse, in the absence of other algorithms that help discover content, it would be beneficial for the bigger picture to make boosting more accessible.
